<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NaturalSumWindow</class>
 <widget class="QMainWindow" name="NaturalSumWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>61</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>back</string>
    </property>
   </widget>
   <widget class="QPushButton" name="result">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>100</y>
      <width>112</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>result</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="result_out">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>751</width>
      <height>241</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>221</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Предыдущий результат:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="replace_known_1">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>30</y>
      <width>261</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Вставить предыдыдущий результат</string>
    </property>
   </widget>
   <widget class="QPushButton" name="replace_known_2">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>60</y>
      <width>261</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Вставить предыдущий результат</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>102</x>
      <y>40</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="natural1">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>30</y>
      <width>281</width>
      <height>21</height>
     </rect>
    </property>
    <property name="maxLength">
     <number>999999999</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="natural2">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>60</y>
      <width>281</width>
      <height>21</height>
     </rect>
    </property>
    <property name="maxLength">
     <number>999999999</number>
    </property>
   </widget>
   <widget class="QLabel" name="warning">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>0</y>
      <width>221</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
